src/main/views/components/accordion/template.njk

```diff
@@ -0,0 +1,36 @@
+{% set id = params.id %}
+
+<div class="govuk-accordion {%- if params.classes %} {{ params.classes }}{% endif -%}" data-module="govuk-accordion" id="{{ id }}"
```

Comment:
Confused by all this. Where did it come from?

Reply:
The show/hide sections parts of the accordion weren't translating using the standard accordion component. This was the only way I could get it to work. It came from here https://github.com/hmcts/rpts-frontend/blob/e0e9c8688c684cb036c9a84a06bedaffe1ab2377/src/main/views/govuk/components/accordion/template.njk#L6